src/cmd/login/login.go

```diff
@@ -68,25 +69,59 @@ func login(_ *cobra.Command, _ []string) error {
 			return err
 		}
 		prints.PrintCliStderr("User registered as Otterize user with user ID: %s", user.Id)
-		userId = user.Id
+		meResponse, err = c.MeQueryWithResponse(registerCtxTimeout)
+		if err != nil {
+			return err
+		}
 	} else if err != nil {
 		return err
-	} else {
-		userId = meResponse.JSON200.User.Id
 	}
 
-	// query user to get full user info
-	userResponse, err := c.UserQueryWithResponse(registerCtxTimeout, userId)
-	if err != nil {
-		return err
-	}
-
-	user := lo.FromPtr(userResponse.JSON200)
+	// FIXME: create task to
+	user := meResponse.JSON200.User
 	prints.PrintCliStderr("Logged in as Otterize user %s (%s)", user.Id, user.Email)
-	if user.Organization != nil && user.Organization.Id != "" {
-		prints.PrintCliStderr("User is registered with organization %s", user.Organization.Id)
+	if len(*user.Organizations) != 0 {
+		prints.PrintCliStderr("You belong to the following organizations:")
+		for _, org := range *user.Organizations {
+			if org.Name != nil {
+				prints.PrintCliStderr("ID: %s | Name: %s", org.Id, *org.Name)
+			} else {
+				prints.PrintCliStderr("ID: %s", org.Id)
+			}
+		}
+		if len(*user.Organizations) == 1 {
+			prints.PrintCliStderr("Only 1 organization found - auto-selecting this organization for use. To change, join another organization and log in again.")
+		} else {
```

Comment:
let's not require input from the user here as a standard, IMO it has the potential to create a very annoying UX. I'm thinking of a few options:
1. if there's only a saved selected org ID in the saved config, use that as default
2. add an input cmd arg to allow selecting org ID via command line args
3. Leave the selected org ID empty in this case, and add an `otterize organizations select` command to allow choosing from it later on. This might be relevant anyway since it makes less sense (IMO) to require the user to login again if all they want is to switch org.

Reply:
I chose #1 - it auto-uses the previously selected org if there's one. To switch org, you need to pass --switch-org in that case. But if you've never selected an org, you have to choose an org interactively - otherwise you can't complete the log-in.